internal/pkg/auth/sso/auth_state.go

```diff
@@ -226,7 +222,7 @@ func (s *authState) getOAuthTokenResponse(payload *strings.Reader) (map[string]a
 	return data, nil
 }
 
-func (s *authState) getAuthorizationCodeUrl(ssoProviderConnectionName string) string {
+func (s *authState) getAuthorizationCodeUrl(ssoProviderConnectionName string, isOkta bool) string {
```

Comment:
We wanted to avoid hard-coding a vendor name when originally writing the SSO code -- can we avoid that here?  What if in the future there are more IDPs we support?

Reply:
Due to lots of variability in how IDPs implement the OAuth "standard", the best approach might be to define an interface that each IDP can conform to. I can do this in a follow-up PR!

Reply:
Sounds good, it would be good if you could create a ticket to track that follow-up work so we don't forget about it.  Thanks!